content/en/docs/Getting Started/pipelines.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "Getting Started with Pipelines"
+linkTitle: "Getting Started with Pipelines"
+weight: 1
+description: >
+  Prerequisites, Installation, and Basic Usage
+---
+
+## Prerequisites
+
+* Completed the [Getting Started example](/docs/getting-started/).
+
+## Extending your first CI/CD Workflow with a second Task and a Pipeline
+
+As you learned previously, with Tekton, each operation in your CI/CD workflow becomes a `Step`,
+which is executed with a container image you specify. `Steps` are then
+organized in `Tasks`, which run as a [Kubernetes pod](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/)
+in your cluster. You can further organize `Tasks` into `Pipelines`, which 
+can control the order of execution of several `Tasks`. 
+
+To create a second `Task`, create a Kubernetes object using the Tekton API with
+the kind `Task`. The following YAML file specifies a `Task` with one simple
+`Step`, which prints a `Goodbye World!` message using
+[the official Ubuntu image](https://hub.docker.com/_/ubuntu/):
+
+```yaml
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: goodbye
+spec:
+  steps:
+    - name: goodbye
+      image: ubuntu
+      command:
```

Comment:
NIT: we should promote the `script` syntax here instead of command, something like

```
spec:
  steps:
    - name: goodbye
      image: ubuntu
      script: |
        set -e
        echo "Goodbye World!"
```

Reply:
fixed.